components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.hpp

```diff
@@ -240,7 +240,7 @@ static void rrtmgp_initialize(
   load_cld_lutcoeff(cloud_optics_lw_k, cloud_optics_file_lw);
 
   // initialize kokkos rrtmgp pool allocator
-  const size_t base_ref = 18000;
+  const size_t base_ref = 4000;
```

Comment:
What's base_ref used for?

Reply:
This is a bit hacky. When I was doing the standalone rrtmgp work, I tried to come up with a simple function for the amount of pool memory needed. I found that comparing (ncol * nlay * nlev) to base_ref got me a reasonable estimate, but for some reason, way less is needed for full cases.

Reply:
this is just the "initial" guess for the pool, and then it can be expanded if needed, right?

Reply:
Not sure what you mean. We can change this number in the source code (which reqs a rebuild to take effect). The pool cannot expand at runtime; so if we overflow it, the case will crash.

Reply:
Ah, ok. I thought it was like YAKL, which whenever the pool fills, opens another pool.

Is there any back of the envelope calculation to ensure this size is enough then?

Reply:
@bartgol , I'm not sure YAKL works that way. There's been many times we've had to increase `GATOR_INITIAL_MB`.

I arrived at this number through experimentation on rrtmgp standalone.